numpy/_core/_add_newdocs.py

```diff
@@ -3217,6 +3223,9 @@
     ComplexWarning
         When casting from complex to float or int. To avoid this,
         one should use ``a.real.astype(t)``.
+    ValueError
+        When casting using ``'same_value'`` and the values change or would
+        overflow
```

Comment:
Can we add an example? Could just continue with the one given already,
```
>>> x.astype(int, casting="same_value")
ValueError(...)

>>> x[:2].astype(int, casting="same_value")
array([1, 2])
```

Reply:
done